src/ray/object_manager/object_manager.cc

```diff
@@ -325,14 +256,12 @@ ray::Status ObjectManager::SendObjectHeaders(const ObjectID &object_id,
       buffer_pool_.GetChunk(object_id, data_size, metadata_size, chunk_index);
   ObjectBufferPool::ChunkInfo chunk_info = chunk_status.first;
 
-  if (!chunk_status.second.ok()) {
-    // This is the first thread to invoke GetChunk => Get failed on the
-    // plasma client.
-    // No reference is acquired for this chunk, so no need to release the chunk.
-    // TODO(hme): Retry send here? If so, store RemoteConnectionInfo in SenderConnection.
-    RAY_CHECK_OK(TransferCompleted(TransferQueue::TransferType::SEND));
-    return chunk_status.second;
-  }
+  // If status is not okay, and this is the first chunk to be gotten for this object,
```

Comment:
The comment about "this is the first chunk to be gotten" isn't correct anymore, right? Can we remove that?

Reply:
The comment seems to be correct to me. Can you explain why it's no longer correct?

Reply:
Shouldn't it be "If status is not okay, then return immediately..."? I don't think it matters which chunk of the object it is.